<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>DialogSpace</class>
 <widget class="QDialog" name="DialogSpace">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>560</width>
    <height>233</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>560</width>
    <height>233</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>560</width>
    <height>233</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Spatialisation</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../ressources.qrc">
    <normaloff>:/icones/icon</normaloff>:/icones/icon</iconset>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item>
      <widget class="QFrame" name="frame_3">
       <property name="frameShape">
        <enum>QFrame::Panel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Sunken</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="GraphSpace" name="grapheSpace" native="true">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Minimum" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Motif&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Nombre divisions</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_3">
          <property name="text">
           <string>Etalement (%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Occupation (%)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_5">
          <property name="text">
           <string>Offset (0-100)</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>Renversements</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Maximum" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_3">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <widget class="QComboBox" name="comboMotif">
          <item>
           <property name="text">
            <string>Ascendant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Creux</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Descendant</string>
           </property>
          </item>
          <item>
           <property name="text">
            <string>Pointe</string>
           </property>
          </item>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinNbDivision">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>10</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinEtalement">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinOccupation">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>100</number>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="spinOffset">
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>50</number>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <property name="bottomMargin">
           <number>0</number>
          </property>
          <item>
           <widget class="QCheckBox" name="checkSens">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="checkSens2">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GraphSpace</class>
   <extends>QWidget</extends>
   <header>dialog_space.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../ressources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>DialogSpace</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>DialogSpace</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>comboMotif</sender>
   <signal>currentIndexChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>motifChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>22</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinNbDivision</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>nbDivChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>53</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinEtalement</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>etalementChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>84</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinOccupation</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>occupationChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>115</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>spinOffset</sender>
   <signal>valueChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>offsetChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>370</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>101</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSens</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>sensChanged(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>355</x>
     <y>162</y>
    </hint>
    <hint type="destinationlabel">
     <x>206</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>checkSens2</sender>
   <signal>stateChanged(int)</signal>
   <receiver>DialogSpace</receiver>
   <slot>sens2Changed(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>510</x>
     <y>173</y>
    </hint>
    <hint type="destinationlabel">
     <x>271</x>
     <y>113</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>motifChanged(int)</slot>
  <slot>nbDivChanged(int)</slot>
  <slot>etalementChanged(int)</slot>
  <slot>occupationChanged(int)</slot>
  <slot>offsetChanged(int)</slot>
  <slot>sensChanged(int)</slot>
  <slot>sens2Changed(int)</slot>
 </slots>
</ui>
